### ISSUE TITLE
Invalid Apify fingerprint data causing error, need to add a safeguard logic

### ISSUE THREAD

**Issue by cda2** [daijro/browserforge#17]
Hello,

It seems that there’s an error in the Apify fingerprint data currently used by BrowserForge, and this issue is propagating errors to both BrowserForge and Camoufox.

![image](https://github.com/user-attachments/assets/952ff123-baed-46b4-9e45-f8653dafa316)

When you download the Apify fingerprint data used by BrowserForge, you can see that some incorrect data is included (as shown in the attached image). As a result, errors are occurring when using BrowserForge or Camoufox.
Please refer to [issue #1](https://github.com/daijro/camoufox/issues/124) and [issue #2](https://github.com/daijro/camoufox/issues/125).

Until the upstream issue is resolved, we may need to add a safeguard logic as [mentioned](https://github.com/daijro/camoufox/issues/124#issuecomment-2533343907) in the comments on other related issues.

**Comment by daijro** [comment]
Hello, thanks for bringing this to my attention.

It appears that in the latest model update, the data has been significantly contaminated with pentesting payloads.

Here is a couple that I have found:

SQL injection attempts:
```python
"pixelDepth\":\"24' OR '1'='1' -- \""
"availWidth\":\"1366 AND 1=2 -- \""
```

CLI injection:

```python
"pageYOffset\":\"0&cat /etc/passwd&\""
"availLeft\":\"0&sleep 25.0&\""
"innerHeight\":\"0\\\"|timeout /T 25.0\""
```

---

Needless to say, this is disturbing for any downstream projects using Apify's data or Browserforge. I have reverted back to v2.1.58 of the model (from 2 weeks ago), which I have found to be cleaner.
